src/main/java/org/jboss/ejb/client/AuthenticationContextEJBClientInterceptor.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2021 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.ejb.client;
+
+import static org.jboss.ejb._private.Keys.AUTHENTICATION_CONTEXT_ATTACHMENT_KEY;
+
+import org.jboss.ejb.client.annotation.ClientInterceptorPriority;
+import org.wildfly.common.function.ExceptionSupplier;
+import org.wildfly.security.auth.client.AuthenticationContext;
+
+/**
+ * EJB client interceptor to capture the {@code AuthenticationContext} after any application interceptors.
+ *
+ * @author <a href="mailto:darran.lofthouse@redhat.com">Darran Lofthouse</a>
+ */
+public class AuthenticationContextEJBClientInterceptor implements EJBClientInterceptor {
+
+    public static final int PRIORITY = ClientInterceptorPriority.JBOSS_AFTER + 25;
+
```

Comment:
looks like this field is not used.

Reply:
Sorry yes I am missing a line - will add.